src/Operations.jl

```diff
@@ -643,25 +643,56 @@ function download_artifacts(ctx::Context, pkgs::Vector{PackageSpec}; platform::A
     return download_artifacts(ctx, pkg_roots; platform=platform, verbose=verbose)
 end
 
+"""
+    load_pkg_hook(hook_path::String)
+
+Given a path to a hook file (which should end in a function definition) include
+it into an anonymous module and return that function.
+"""
+function load_pkg_hook(hook_path::String)
+    m = Module(:__anon__)
```

Comment:
Doesn't this need to be `gensym` or something in order not to cause clashes?
```suggestion
    m = Module(gensym())
```